### PR TITLE
Fixed blocking loader, switched default translation to Clear Quran

### PR DESCRIPTION
**Pull request by naveed-ahmad**
- Default translation now is Clear Quran with footnotes( we'll remove simple Clear Quran with next build )
- Fixed loaders
- Fixed  ayat ul kursi https://github.com/quran/quran.com-frontend/issues/927
- Fixed jump to ayah beyond 10 https://github.com/quran/quran.com-frontend/issues/902 and https://github.com/quran/quran.com-frontend/issues/899
- Fixed too many redirect https://github.com/quran/quran.com-frontend/issues/885
- Scrolling issue https://github.com/quran/quran.com-frontend/issues/872
- https://github.com/quran/quran.com-frontend/issues/873
- https://github.com/quran/quran.com-frontend/issues/816
- https://github.com/quran/quran.com-frontend/issues/753

### Checklist
- [ ] Unit tests written
- [x] Manually tested
- [x] Prettier & ESLint were run
- [x] New dependencies are included in `package-lock.json`

### CODE REVIEW

**Comment by naveed-ahmad** [comment]
rebuild

**Comment by naveed-ahmad** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33012

**Comment by mmahalwy** [comment]
Sweet! Let me know when to review!

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33016

**Comment by naveed-ahmad** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33017

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33019

File: src/components/Audioplayer/ScrollButton/index.js [hunk 1]
```diff
@@ -1,49 +1,45 @@
...
+    <StyledPopover
+      id="scroll-toggle-popoverr"
+      title={
+        <LocaleFormattedMessage
+          id="player.scrollButtonTip"
+          defaultMessage="Automatically scrolls to the currently playing ayah on transitions..."
+        />
+      }
+    >
+      {'  '}
```
Comment (AlaaAttya) [inline src/components/Audioplayer/ScrollButton/index.js]: better to style it instead of using empty spaces

Comment (naveed-ahmad) [reply src/components/Audioplayer/ScrollButton/index.js]: Toggle is already centered align. Forget to remove this :) thanks for reminding.

File: src/containers/Chapter/index.js [hunk 2]
```diff
@@ -53,27 +53,29 @@ const TopOptions = Loadable({
...
-      if (start > chapter.versesCount || isNaN(start)) {
-        return actions.push.push('/error/invalid-verse-range');
-      }
-
-      return false;
-    }
-
-    return false;
-  }
+  //
```
[the remark this answers is not in the document]
Comment (naveed-ahmad) [reply src/containers/Chapter/index.js]: in next PR inshAllah.

**Pull request merged**